dato/js/recview.js

```diff
@@ -0,0 +1,158 @@
+$(document).ready( function(event){
```

Comment:
Strange. Where is that `picture-canvsas.html` used? I don't see any other references to it in the userguide repo (other files including it as, say, an iframe, nor links to it).

The jquery dependency seems to be coming from [here](https://github.com/dato-code/userguide/blob/74b334f53e556c044283e1a7c84868360ff5e598/book.json#L18) so I think we're safe (it should be guaranteed to be on every page, as per our config of GitBook).